workflow/hosts/azurepw.yaml

```
@@ -26,5 +26,8 @@ MAKE_NSSTBUFR: 'NO'
 MAKE_ACFTBUFR: 'NO'
 DO_TRACKER: 'NO'
 DO_GENESIS: 'NO'
-DO_METP: 'NO'
+DO_METP: 'NO' 
+DO_FETCH_HPSS: 'NO' 
+SUPPORT_WAVES: 'NO'
 SUPPORTED_RESOLUTIONS: ['C48', 'C96', 'C384', 'C768'] # TODO: Test and support all cubed-sphere resolutions.
+
```

`SUPPORT_WAVES` isn't used anymore.

```suggestion
SUPPORTED_RESOLUTIONS: ['C48', 'C96', 'C384', 'C768'] # TODO: Test and support all cubed-sphere resolutions.
```